### PR TITLE
Refactor FXIOS-4594 [v108] Improve displayed items for JumpBackIn section on iPad

### PR DESCRIPTION
**Pull request by thatswinnie**
[Jira](https://mozilla-hub.atlassian.net/browse/FXIOS-4594)
#11339

### CODE REVIEW

**Review by nishant2718 — APPROVED** [review]

File: Client/Frontend/Home/JumpBackIn/JumpBackInSectionLayout.swift [hunk 1]
```diff
@@ -16,15 +16,21 @@ enum JumpBackInSectionLayout: Equatable {
...
+    case mediumWithSyncedTab // synced tab is displayed first and then jumpBackIn
+
     // The width dimension of a cell / group; takes into account how many groups will be displayed
     var widthDimension: NSCollectionLayoutDimension {
         switch self {
         case .compactJumpBackIn, .compactSyncedTab, .compactJumpBackInAndSyncedTab:
             // When the trailing inset will be handled by the section layout, this can be set to .fractionalWidth(1)
             return .fractionalWidth(0.95)
         case .regular, .regularWithSyncedTab:
+            return .fractionalWidth(7.66/24)
```
Comment (polymathruby) [inline Client/Frontend/Home/JumpBackIn/JumpBackInSectionLayout.swift:30]: Would it make sense to define these fractions somewhere?

File: Client/Frontend/Home/JumpBackIn/JumpBackInViewModel.swift [hunk 2]
```diff
@@ -145,6 +150,22 @@ class JumpBackInViewModel: FeatureFlaggable {
...
+        guard let window = UIWindow.keyWindow else { return false }
+
+        return window.frame.width != window.screen.bounds.width && window.frame.width != window.screen.bounds.height
+    }
+
+    private func isPadInLandscapeSplit(split: CGFloat,
+                                       isPortrait: Bool = UIWindow.isPortrait,
+                                       device: UIUserInterfaceIdiom = UIDevice.current.userInterfaceIdiom) -> Bool {
+        guard device == .pad, !isPortrait, isMultitasking, let window = UIWindow.keyWindow
+        else { return false }
```
Comment (polymathruby) [inline Client/Frontend/Home/JumpBackIn/JumpBackInViewModel.swift:164]:
```suggestion
        guard device == .pad, 
                   !isPortrait, 
                   isMultitasking, 
                   let window = UIWindow.keyWindow
        else { return false }
```

This is a nit. Feel free to ignore.

Comment (polymathruby) [reply Client/Frontend/Home/JumpBackIn/JumpBackInViewModel.swift:164]: GH HATES MY ALIGNMONSTS!

**Review by polymathruby — APPROVED** [review]

File: Client/Frontend/Home/JumpBackIn/JumpBackInSectionLayout.swift [hunk 3]
```diff
@@ -16,15 +16,21 @@ enum JumpBackInSectionLayout: Equatable {
...
+        case .medium, .mediumWithSyncedTab:
             return UIDevice.current.userInterfaceIdiom == .pad ?
-                .fractionalWidth(7.66/24) : .fractionalWidth(7.8/16) // iPad or iPhone in landscape
+                .fractionalWidth(7.66/16) : .fractionalWidth(7.8/16) // iPad or iPhone in landscape
```
Comment (yoanarios) [inline Client/Frontend/Home/JumpBackIn/JumpBackInSectionLayout.swift:33]: nit: Can you add a comment on how to get to those values

**Review by yoanarios — APPROVED** [review]

**Comment by thatswinnie** [comment]
Build passed
Screen Shot 2022-10-28 at 10 48 53

**Pull request merged**